integration/spark-common-test/src/test/scala/org/apache/carbondata/spark/testsuite/dataload/TestLoadDataWithHiveSyntaxDefaultFormat.scala

```diff
@@ -687,6 +688,48 @@ class TestLoadDataWithHiveSyntaxDefaultFormat extends QueryTest with BeforeAndAf
     checkAnswer(sql("select salary from double_test limit 1"),Row(7.756787654567891E23))
   }
 
+  test("test table with specified table path") {
+    val path = "./source"
+    sql("drop table if exists table_path_test")
+    sql(
+      "CREATE table table_path_test (empno string, salary double) STORED BY 'carbondata' " +
+      s"TBLPROPERTIES('table_path'='$path')"
```

Comment:
We should not take path from tableproperties . Better use standard way of specifying the path. For both spark datasource and hive ddl uses [LOCATION path] to take tablePath. Better use the same.

Reply:
ok